.travis.yml

```diff
@@ -3,8 +3,8 @@ cache: npm
 notifications:
   email: false
 node_js:
-  - 10.0.0
-  - 12
+  - lts/dubnium
+  - lts/erbium
```

Comment:
I don't know if these are valid versions for config, kicking CI to see

Reply:
It works https://travis-ci.org/github/testing-library/dom-testing-library/builds/673703620